.github/workflows/image-scan.yml

```diff
@@ -148,6 +148,7 @@ jobs:
           summaryForScanner() {
             for resultJSONFile in $1; do
               image=$(jq -r '.image' $resultJSONFile)
+              detailedArtifactName=$(echo $image | sed 's/[^a-zA-Z0-9.-]/-/g')-scan-detailed-result.json
```

Comment:
No such artifact exists. The artifact name ends either with `-lw-scan-detailed-result.json` or `-aqua-scan-detailed-result.json`